installer/create-cluster-eks.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+
+# Copyright (C) 2016-2019 Lightbend Inc. <https://www.lightbend.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Usage:
+# create-cluster-eks.sh [EKS-CLUSTER-NAME] [AWS_DEFAULT_REGION]
+if [ $# -ne 2 ]; then
+  echo "Not enough arguments supplied"
+  echo "Usage: create-cluster-eks.sh [EKS-CLUSTER-NAME] [AWS_DEFAULT_REGION]"
+  exit 1
+fi
+
+export EKS_CLUSTER_NAME=$1
+export AWS_DEFAULT_REGION=$2
+
+# Create cluster
+# https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html
+ZONE_1A=a
+ZONE_1B=b
+ZONE_1D=d
+eksctl create cluster \
```

Comment:
From what I read here: https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html a yaml file could be used to describe the cluster resources. I think this way would be easier for the user to modify if needed but requires more work to generate it since it depends on the region etc.